<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Report_management</class>
 <widget class="QWidget" name="Report_management">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1035</width>
    <height>706</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Report_management</string>
  </property>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1035</width>
     <height>706</height>
    </rect>
   </property>
   <property name="tabPosition">
    <enum>QTabWidget::North</enum>
   </property>
   <property name="tabShape">
    <enum>QTabWidget::Rounded</enum>
   </property>
   <property name="currentIndex">
    <number>4</number>
   </property>
   <property name="documentMode">
    <bool>false</bool>
   </property>
   <property name="tabsClosable">
    <bool>false</bool>
   </property>
   <property name="movable">
    <bool>true</bool>
   </property>
   <property name="tabBarAutoHide">
    <bool>false</bool>
   </property>
   <widget class="QWidget" name="tab">
    <attribute name="title">
     <string>预定客人报表</string>
    </attribute>
    <widget class="QTableView" name="tableView_2">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>50</y>
       <width>740</width>
       <height>580</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_10">
     <property name="geometry">
      <rect>
       <x>850</x>
       <y>280</y>
       <width>110</width>
       <height>70</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_2">
    <attribute name="title">
     <string>入住客人报表</string>
    </attribute>
    <widget class="QTableView" name="tableView_3">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>50</y>
       <width>740</width>
       <height>580</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_11">
     <property name="geometry">
      <rect>
       <x>850</x>
       <y>280</y>
       <width>110</width>
       <height>70</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_3">
    <attribute name="title">
     <string>当日客人预定报表</string>
    </attribute>
    <widget class="QTableView" name="tableView_4">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>50</y>
       <width>740</width>
       <height>580</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_12">
     <property name="geometry">
      <rect>
       <x>850</x>
       <y>280</y>
       <width>110</width>
       <height>70</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QDateEdit" name="dateEdit_5">
     <property name="geometry">
      <rect>
       <x>840</x>
       <y>120</y>
       <width>110</width>
       <height>22</height>
      </rect>
     </property>
     <property name="displayFormat">
      <string>yyyy/MM/dd</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_4">
     <property name="geometry">
      <rect>
       <x>840</x>
       <y>80</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>显示当日</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_4">
    <attribute name="title">
     <string>当日客人入住报表</string>
    </attribute>
    <widget class="QTableView" name="tableView_5">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>50</y>
       <width>740</width>
       <height>580</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_13">
     <property name="geometry">
      <rect>
       <x>850</x>
       <y>280</y>
       <width>110</width>
       <height>70</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_2">
     <property name="geometry">
      <rect>
       <x>840</x>
       <y>80</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>显示当日</string>
     </property>
    </widget>
    <widget class="QDateEdit" name="dateEdit_3">
     <property name="geometry">
      <rect>
       <x>840</x>
       <y>120</y>
       <width>110</width>
       <height>22</height>
      </rect>
     </property>
     <property name="displayFormat">
      <string>yyyy/MM/dd</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_5">
    <attribute name="title">
     <string>当日离店客人报表</string>
    </attribute>
    <widget class="QTableView" name="tableView_6">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>50</y>
       <width>740</width>
       <height>580</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_14">
     <property name="geometry">
      <rect>
       <x>850</x>
       <y>280</y>
       <width>110</width>
       <height>70</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QDateEdit" name="dateEdit_4">
     <property name="geometry">
      <rect>
       <x>840</x>
       <y>120</y>
       <width>110</width>
       <height>22</height>
      </rect>
     </property>
     <property name="displayFormat">
      <string>yyyy/MM/dd</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_3">
     <property name="geometry">
      <rect>
       <x>840</x>
       <y>80</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>显示当日</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_6">
    <attribute name="title">
     <string>当日财务进帐报表</string>
    </attribute>
    <widget class="QPushButton" name="pushButton_15">
     <property name="geometry">
      <rect>
       <x>850</x>
       <y>280</y>
       <width>110</width>
       <height>70</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QChartView" name="chartView" native="true">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>50</y>
       <width>740</width>
       <height>580</height>
      </rect>
     </property>
    </widget>
    <widget class="QDateEdit" name="dateEdit">
     <property name="geometry">
      <rect>
       <x>890</x>
       <y>171</y>
       <width>101</width>
       <height>20</height>
      </rect>
     </property>
     <property name="dateTime">
      <datetime>
       <hour>0</hour>
       <minute>0</minute>
       <second>0</second>
       <year>2021</year>
       <month>1</month>
       <day>18</day>
      </datetime>
     </property>
     <property name="displayFormat">
      <string>yyyy/MM/dd</string>
     </property>
     <property name="date">
      <date>
       <year>2021</year>
       <month>1</month>
       <day>18</day>
      </date>
     </property>
    </widget>
    <widget class="QDateEdit" name="dateEdit_2">
     <property name="geometry">
      <rect>
       <x>890</x>
       <y>128</y>
       <width>101</width>
       <height>20</height>
      </rect>
     </property>
     <property name="dateTime">
      <datetime>
       <hour>0</hour>
       <minute>0</minute>
       <second>0</second>
       <year>2021</year>
       <month>1</month>
       <day>9</day>
      </datetime>
     </property>
     <property name="time">
      <time>
       <hour>0</hour>
       <minute>0</minute>
       <second>0</second>
      </time>
     </property>
     <property name="displayFormat">
      <string>yyyy/MM/dd</string>
     </property>
     <property name="date">
      <date>
       <year>2021</year>
       <month>1</month>
       <day>9</day>
      </date>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>829</x>
       <y>128</y>
       <width>48</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>开始日期</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>829</x>
       <y>171</y>
       <width>48</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>结束日期</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton">
     <property name="geometry">
      <rect>
       <x>820</x>
       <y>80</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>刷新</string>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QChartView</class>
   <extends>QWidget</extends>
   <header location="global">qchartview.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
